### PR TITLE
vorstellungsseite schul cloud team

### PR DESCRIPTION
**Pull request by Langleu**
- [x] missing texts
 4 pictures missing, will be added as soon as they are there.
Can still be merged into master.

### CODE REVIEW

File: controllers/team.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const authHelper = require('../helpers/authentication');
+
+// Partners
```
Comment (pinussilvestrus) [inline controllers/team.js]: vielleicht, ganz vielleicht, könnte man dieses gecopy-paste noch wegmachen 😛

Comment (Langleu) [reply controllers/team.js]: war ja noch WIP 😜

**Comment by janrenz** [comment]
screen shot 2018-02-01 at 10 08 01

**Comment by Langleu** [comment]
@janrenz 
a) I will try it out, but problem is that some people have too much text and then the header disappears entirely, but the footer will always be visible.
b) the reason why the nav is overlapping is, that the nav just lies over everything and the content is just moved 240px to the right. Fixed it by adding the padding to the left, same reason why the footer disappears currently in the master branch and your pr as well.

**Comment by Langleu** [comment]
alle RFCs eingebaut .. @janrenz @pinussilvestrus

**Comment by Langleu** [comment]
so dürft wieder

**Comment by Langleu** [comment]
[Vorstellungsseite Schul-Cloud-Team](https://trello.com/c/YVgmn1MG/871-vorstellungsseite-schul-cloud-team)

**Pull request merged**